app.js

```diff
@@ -1,9 +1,29 @@
 'use strict';
 
 const herokuCluster = require('@quarterto/heroku-cluster');
+const https = require('https');
+const denodeify = require('denodeify');
```

Comment:
we already have `@quarterto/promisify` as a dependency which is the same thing but written by me 😉